packages/expo-cli/src/commands/apply/configureIOSProjectAsync.ts

```diff
@@ -0,0 +1,90 @@
+    .normalize('NFD')
+    .replace(/[\u0300-\u036f]/g, '');
+}
+
+// TODO: it's silly and kind of fragile that we look at app config to determine the
+// ios project paths
+// * Overall this function needs to be revamped, just a placeholder for now!
+function getIOSPaths(projectRoot: string) {
+  const { exp } = getConfig(projectRoot, { skipSDKVersionRequirement: true });
+
+  let projectName = exp.name;
+  if (!projectName) {
+    throw new Error('Need a name ;O');
+  }
+
+  const iosProjectDirectory = path.join(projectRoot, 'ios', sanitizedName(projectName));
+  const iconPath = path.join(iosProjectDirectory, 'Assets.xcassets', 'AppIcon.appiconset');
+
+  return {
+    projectName,
+    iosProjectDirectory,
+    iconPath,
+  };
+}
+
+async function modifyEntitlementsPlistAsync(projectRoot: string, callback: (plist: any) => any) {
+  let entitlementsPath = IOSConfig.Entitlements.getEntitlementsPath(projectRoot);
+  let directory = path.dirname(entitlementsPath);
+  let filename = path.basename(entitlementsPath, 'plist');
+  await IosPlist.modifyAsync(directory, filename, callback);
+  await IosPlist.cleanBackupAsync(directory, filename, false);
+}
+
+async function modifyInfoPlistAsync(projectRoot: string, callback: (plist: any) => any) {
+  const { iosProjectDirectory } = getIOSPaths(projectRoot);
+  await IosPlist.modifyAsync(iosProjectDirectory, 'Info', callback);
+  await IosPlist.cleanBackupAsync(iosProjectDirectory, 'Info', false);
+}
+
+export default async function configureIOSProjectAsync(projectRoot: string) {
+  const { exp } = getConfig(projectRoot, { skipSDKVersionRequirement: true });
+
+  IOSConfig.BundleIdenitifer.setBundleIdentifierForPbxproj(projectRoot, exp.ios!.bundleIdentifier!);
+
+  // Configure the Info.plist
+  await modifyInfoPlistAsync(projectRoot, infoPlist => {
+    infoPlist = IOSConfig.CustomInfoPlistEntries.setCustomInfoPlistEntries(exp, infoPlist);
+    infoPlist = IOSConfig.Name.setDisplayName(exp, infoPlist);
+    infoPlist = IOSConfig.Scheme.setScheme(exp, infoPlist);
+    infoPlist = IOSConfig.Version.setVersion(exp, infoPlist);
+    infoPlist = IOSConfig.Version.setBuildNumber(exp, infoPlist);
+    infoPlist = IOSConfig.DeviceFamily.setDeviceFamily(exp, infoPlist);
+    infoPlist = IOSConfig.RequiresFullScreen.setRequiresFullScreen(exp, infoPlist);
+    infoPlist = IOSConfig.UserInterfaceStyle.setUserInterfaceStyle(exp, infoPlist);
+    infoPlist = IOSConfig.Branch.setBranchApiKey(exp, infoPlist);
+    infoPlist = IOSConfig.UsesNonExemptEncryption.setUsesNonExemptEncryption(exp, infoPlist);
+    infoPlist = IOSConfig.Facebook.setFacebookConfig(exp, infoPlist);
+
+    return infoPlist;
+  });
+
+  // Configure entitlements/capabilities
+  await modifyEntitlementsPlistAsync(projectRoot, entitlementsPlist => {
+    // TODO: We don't have a mechanism for getting the apple team id here yet
+    entitlementsPlist = IOSConfig.Entitlements.setICloudEntitlement(
+      exp,
+      'TODO-GET-APPLE-TEAM-ID',
```

Comment:
Probably want to use something like [appleApi/authenticate](https://github.com/expo/expo-cli/blob/c4b46396c3cb947fdcc5f17a3b587b71815f7ae3/packages/expo-cli/src/appleApi/authenticate.ts#L122)

Reply:
yeah, going to defer this one until after release to follow up with